target/linux/ath79/dts/qca9531_letv_lba-047-ch.dts

```diff
@@ -91,6 +91,7 @@
 			};
 
 			partition@40000 {
+				compatible = "u-boot,env";
 				label = "u-boot-env";
```

Comment:
It seems that you convert them by a script. This device doesn't have uboot-env config. Please also check other devices.